mono/mini/mini-runtime.c

```diff
@@ -4749,6 +4771,7 @@ mini_cleanup (MonoDomain *domain)
 {
 	print_jit_stats ();
 	mono_counters_dump (MONO_COUNTER_SECTION_MASK | MONO_COUNTER_MONOTONIC, stdout);
+	mono_stats_method_name_clean();
```

Comment:
Don't worry about freeing it here. If you were to free it, keep in mind that calling `g_free` on a pointer will not change the value of the variable that holds that pointer, so you would have a dangling reference to freed memory. A later attempt to access that would cause a crash.